platform/ui/src/components/Button/Button.tsx

```diff
@@ -128,7 +128,11 @@ Button.propTypes = {
   /** Callback to be called when the button is clicked  */
   onClick: PropTypes.func.isRequired,
   /** Button size  */
-  size: PropTypes.oneOf([ButtonEnums.size.medium, ButtonEnums.size.small]),
+  size: PropTypes.oneOf([
```

Comment:
we don't have @ohif/ui anymore, this file should not be touched as it is legacy, use the ui-next button